models/pp_cond_exp_mc_urbanczik.h

```diff
@@ -132,6 +132,50 @@ the receptor types given in the receptor_types entry of the state dictionary.
 Note that in contrast to the single-compartment models, all
 synaptic weights must be positive numbers!
 
+-------------------------------------------
+Multicompartment models and synaptic delays
+-------------------------------------------
+
+Note that in case of multicompartment models which model the dendrite
```

Comment:
Could we use a synonym of the verb "model", e.g., "represent"?

Reply:
Okay, I replaced it by "represent".